pkg/controller/podautoscaler/horizontal.go

```diff
@@ -50,8 +51,8 @@ const (
 )
 
 type HorizontalController struct {
-	scaleNamespacer unversionedextensions.ScalesGetter
-	hpaNamespacer   unversionedextensions.HorizontalPodAutoscalersGetter
+	scaleClient   *scaleclient.Client
```

Comment:
@liggitt What's the point in doing that? scaleclient.Client has only one implementation.